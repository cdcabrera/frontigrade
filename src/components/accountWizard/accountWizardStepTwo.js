import React from 'react';

const AccountWizardStepTwo = () => <div>Lorem ipsum step two.</div>;

AccountWizardStepTwo.propTypes = {};

export { AccountWizardStepTwo as default, AccountWizardStepTwo };
