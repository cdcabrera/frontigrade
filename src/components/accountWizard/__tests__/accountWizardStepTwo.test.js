import React from 'react';
import { mount } from 'enzyme';
import { AccountWizardStepTwo } from '../accountWizardStepTwo';

describe('AccountWizardStepTwo Component', () => {
  it('should render', () => {
    const props = {};
    const component = mount(<AccountWizardStepTwo {...props} />);

    expect(component.render()).toMatchSnapshot();
  });
});
