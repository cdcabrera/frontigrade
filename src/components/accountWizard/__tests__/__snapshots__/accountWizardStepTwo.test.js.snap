// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountWizardStepTwo Component should render 1`] = `
<div>
  Lorem ipsum step two.
</div>
`;
